packages/neos-ui/src/manifest.js

```diff
@@ -270,6 +270,7 @@ manifest('main', {}, globalRegistry => {
         const contentElement = (new DOMParser())
             .parseFromString(renderedContent, 'text/html')
             .querySelector(`[data-__neos-node-contextpath="${contextPath}"]`);
+        const fusionPath = contentElement.dataset.__neosFusionPath;
```

Comment:
contentElement might be NULL, see check 3 lines below. So move this after this ;)